contracts/interfaces/ICustomErrors.sol

```diff
@@ -21,7 +21,7 @@ interface ICustomErrors {
 
   error RequiresOwner(address who);
 
-  error InsuficientFunds(uint256 missing);
```

Comment:
It's so good to have a native English speaker in the team...
Good eyes and thanks for being attentive to details!

Reply:
hehe 😅